Microsoft.Toolkit.Uwp.UI.Controls/ColorPickerButton/ColorToDisplayNameConverter.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Windows.UI;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Media;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// Gets the approximated display name for the color.
+    /// </summary>
+    public class ColorToDisplayNameConverter : IValueConverter
```

Comment:
Do we not have this somewhere already? Wondering if we want these converters in our base package where the rest of the Converters life? In the case of the other one that may be more specific to the picker itself, maybe it's in a different sub-namespace so it doesn't show-up or we use the Browsable attribute?

Reply:
I'm not sure if the ColorToDisplayNameConverter  already exists -- I've never seen it before. I put all the converters with the control for now but expected you would let me know if they need to move. I'm happy to put them wherever.

Anyone that wants to re-template the ColorPickerButton will need access to these converters. I would be more inclined to put them in a sub-namespace although I don't see any harm in having some extra ones show up. They are all fairly 'general-purpose' except for ColorToColorShadeConverter.

Reply:
> I'm not sure if the ColorToDisplayNameConverter already exists

I did a search of any usages of the underlying WinRT .ToDisplayName() method and it returned no results. This converter does not already exist.

Reply:
Ended up moving ColorToDisplayNameConverter with the other general-purpose converters. The other converters are now in a sub-namespace specifically for the color picker.